features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/AndroidTracer.kt

```diff
@@ -312,13 +313,13 @@ class AndroidTracer internal constructor(
 
         /**
          * Helper method to attach an error message to a specific Span.
-         * The error message will be added to the provided Span as a standard Error Tag.
+         * The error message will be logged with ERROR severity and can be seen in logs attached to the span.
          * @param span the active Span
          * @param message the error message you want to attach
          */
         @JvmStatic
         fun logErrorMessage(span: Span, message: String) {
-            val fieldsMap = mapOf(Fields.MESSAGE to message)
+            val fieldsMap = mapOf(Fields.MESSAGE to message, "severity" to Log.ERROR)
```

Comment:
we need to call this either `level` (user in codebase) or `status` (facet name) and also extract it as a constant.